src/main/java/org/influxdb/dto/Point.java

```diff
@@ -385,4 +387,51 @@ private StringBuilder formatedTime() {
     return sb;
   }
 
+  public Point tag(final String tagName, final String value) {
+
```

Comment:
Is there a specific tool you use to format code against a predetermined style configuration?

Despite different perspectives, it's easier to conform if it's easy to do and well known.